src/index.js

```diff
@@ -5,6 +5,47 @@ const getTop = function(element, start) {
   return element.getBoundingClientRect().top + start;
 };
 
+// we use requestAnimationFrame to be called by the browser before every repaint
+const requestAnimationFrame = window.requestAnimationFrame ||
+window.mozRequestAnimationFrame || window.webkitRequestAnimationFrame ||
+function(fn) {
+  window.setTimeout(fn, 16);
+};
```

Comment:
I think it would be fine for SSR if this statement was moved inside the `smoothScroll` function below.

Reply:
Happy to make this change if this is all that's what's required to get this merged!